code/modules/mob/new_player/preferences_setup.dm

```diff
@@ -208,7 +208,7 @@
 
 /datum/preferences/proc/random_character()
 	gender = pick(MALE, FEMALE)
-	var/datum/species/S = GLOB.all_species[DEFAULT_SPECIES]
+	var/datum/species/S = GLOB.roundstart_species[DEFAULT_SPECIES]
```

Comment:
this makes a random character, meaning this should be what the datum/preferences species should be, so it picks the species you are
```suggestion
	var/datum/species/S = GLOB.all_species[species]
```